src/EditorFeatures/VisualBasicTest/CodeActions/MoveType/MoveTypeTests.MoveToNewFile.vb

```diff
@@ -63,14 +63,14 @@ End Class
 "
             Dim codeAfterMove =
 "
-Public Partial Class Class1
+Partial Public Class Class1
```

Comment:
It was surprising to me too that partial is preferred to be first, but I'm not a VB dev. I wonder though if it would be possible to change at this point given that the IDE has literally rewritten the code if you used the opposite order until now, and now it started doing the opposite. I'm guessing that 99% current VB code uses what the IDE enforced and now they would open a file, navigate to a location and have it all messed up.